Main.java

```diff
@@ -1,15 +1,23 @@
-/**
- *
- * @author 
- */
+import java.util.Scanner;
+
 public class Main {
 
   /**
-   * The method that is executed when you hit the run button.
-   * @param args the command line arguments
+   * this code will tell you if the number you inputed is even or odd
+   * @author Isaac
    */
   public static void main(String[] args) {
     // TODO code application logic here
-    
-  }
-}
+    Scanner input = new Scanner(System.in);
+     System.out.println("Please enter an integer.");
+     int number = input.nextInt();
+     int numberRemainder = number % 2;
+     if (numberRemainder ==1) { 
+       System.out.println (+ number + " is an odd number." );
+       }  if (numberRemainder ==0) {
+         System.out.println (+ number + " is an even number.");
+       }
+     }
```

Comment:
Watch your indentation here. IT kind of got away from you. Every part of the code inside the main method should pretty well be at the same indentation level. If it is inside of the if statement, it is one extra tab in. As soon as that if statement finished, we go back to the default level.

```
if(condition){  
     /do stuff

} else if (another condition){
   / do other stuff


}
System.out.println("Tada!");
```